packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -770,23 +770,22 @@ VirtualListBase.displayName = 'VirtualListBase';
 const VirtualListBaseNative = VirtualListBaseFactory(Native);
 VirtualListBaseNative.displayName = 'VirtualListBaseNative';
 
-const ScrollableVirtualList = ({role, ...rest}) => ( // eslint-disable-line react/jsx-no-bind
+const ScrollableVirtualList = ({...rest}) => ( // eslint-disable-line react/jsx-no-bind
```

Comment:
Please define `props` instead of `{...rest}`.

Reply:
I missed that point. I changed from 'rest' to 'props'.